content/en/blog/_posts/2019-06-29-some-apis-are-being-deprecated.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: blog
+title: 'Some APIs Are Being Deprecated: Here's What You Need To Know
```

Comment:
I think this is missing a trailing '

Reply:
To avoid the passive voice, I'd try something like:
> APIs Deprecating Soon: Here's What You Need to Know